packages/migration/src/tests/mods/utils/index.ts

```diff
@@ -15,7 +15,7 @@ const fileSystem: any = {};
 async function recordFile(filename: string): Promise<void> {
   const contents = await readFileAsync(filename);
   const trimmedName = filename.substring(fixturesRoot.length);
-  const parts = path.dirname(trimmedName).split(path.sep);
+  const parts = path.dirname(trimmedName).split('/'); // Intentionally not using path.sep, as node is giving '/' for separator instead of os-correct version.
```

Comment:
Umm....per the docs, it should be OS-correct?? https://nodejs.org/api/path.html#path_path_sep

Reply:
Well, `path.sep` is correct, but filename globbing returns os-independent pathnames so it ends up being moot.

Reply:
Got it. In that case can you update the comment?